lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -364,6 +364,16 @@ static SILValue reapplyFunctionConversion(
     // function.
     return innerNewFunc;
   }
+  // begin_borrow
+  if (auto *bbi = dyn_cast<BeginBorrowInst>(oldConvertedFunc)) {
+    auto innerNewFunc = reapplyFunctionConversion(
```

Comment:
You can just return it here. Same for line 359.

Reply:
Done, thanks.